torch/distributed/fsdp/_init_utils.py

```diff
@@ -24,6 +24,7 @@
 import torch.distributed.fsdp._traversal_utils as traversal_utils
 import torch.distributed.fsdp.fully_sharded_data_parallel as fsdp_file
 import torch.nn as nn
+from torch import Tensor
```

Comment:
nit: If we do this, I feel like we should replace all `torch.Tensor` in the file with `Tensor`, or else it gets fragmented.

Maybe just use `torch.Tensor` on L857?